controller/eventController.js

```diff
@@ -65,7 +75,9 @@ const getAll = (req, res) => {
                 //First we extract the event id's from invites
                 let eventIDs = [];
                 invites.forEach(invite => {
```

Comment:
You could use the built-in `filter` function present in arrays maybe chained with the buil-in `map` function maybe?